### PR TITLE
Shifted every file from using memcache to use bsServer

### PR DESCRIPTION
**Pull request by ultrainstinct30**

### CODE REVIEW

**Comment by thebhatman** [comment]
This branch also has merge conflicts. Resolve them.

**Comment by thebhatman** [comment]
Also since your PR for tactics/Goalie.py and run_goalie.py has been merged, make this PR have changes in all other files except these two files.

File: tactics/Goalie.py [hunk 1]
```diff
@@ -88,7 +88,10 @@ def add_kub(self,kub):
 		self.kub = kub
 
 	def peace_out(self):
+<<<<<<< HEAD
 		# state = shared.get('state')
+=======
+>>>>>>> fsm
```
Comment (thebhatman) [inline tactics/Goalie.py]: You have to remove this.

**Comment by saurabhkgp21** [comment]
@thebhatman ask any first year to finish this up

File: tactics/Goalie.py (hunk 1 above)
Comment (thebhatman) [reply tactics/Goalie.py]: In tactics/Goalie.py, This has to be removed. This is due to the merge conflict.

**Pull request merged**